jetty9/src/main/docker/Dockerfile

```diff
@@ -39,8 +39,9 @@ RUN mkdir webapps $TMPDIR \
  && chown -R jetty:jetty $JETTY_BASE $TMPDIR
 
 # Start Jetty directly
-COPY app-env.bash /
-RUN chmod +rx /app-env.bash
+COPY setup-env-ext.bash /tmp/
+RUN cat /tmp/setup-env-ext.bash >> /setup-env.bash \
```

Comment:
Would `#/bin/bash` at the top of `setup-env.bash` throw it off? That's the reason I kind of preferred appeding `source /setup-env-ext.bash`.